Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierVatQuestion.cshtml

```diff
@@ -40,9 +41,18 @@
                                         <!--This is 9 numbers, sometimes with 'GB' at the start, for example 123456789 or GB123456789.-->You can find it
                                         on your VAT registration certificate.
                                     </div>
-                                    <input class="govuk-input govuk-!-width-one-third @(vatRegHasError ? "govuk-form-group--error" : "")"
-                                           id="vatNumber" name="VatNumber" spellcheck="false" autocomplete="false" aria-describedby="vat-registration-number-hint"
-                                           value="@(Model.HasVatNumber == true ? Model.VatNumber : "")">
+
+                                    @if (vatRegHasError)
+                                    {
+                                        <p id="organisation-vat-error" class="govuk-error-message">
+                                            <span class="govuk-visually-hidden">@StaticTextResource.Global_Error:</span>
+                                            @Html.ValidationMessageFor(m => m.VatNumber)
+                                        </p>
+                                    }
+
+                                    <input class="govuk-input govuk-!-width-one-third @(vatRegHasError ? "govuk-input--error" : "")"
+                                           id="@nameof(Model.VatNumber)" name="@nameof(Model.VatNumber)" spellcheck="false" autocomplete="false" aria-describedby="vat-registration-number-hint"
+                                           value="@(Model.HasVatNumber == true ? Model.VatNumber : "")" @(vatRegHasError ? "aria-describedby=organisation-vat-error" : "")>
```

Comment:
the input has two aria-describedby attributes, but should only have one with space separated values. So it'd be like `aria-describedby="vat-registration-number-hint"` when there's no error and `aria-describedby="vat-registration-number-hint organisation-vat-error"` when there's an error

Reply:
@andymantell done.